anthos-bm-gcp-terraform/resources/init_vm.sh

```diff
@@ -163,7 +163,7 @@ function __setup_kubctl__ () {
 ##############################################################################
 function __setup_bmctl__ () {
   mkdir baremetal && cd baremetal || return
-  gsutil cp gs://anthos-baremetal-release/bmctl/1.11.1/linux-amd64/bmctl .
+  gsutil cp gs://anthos-baremetal-release/bmctl/1.12.0/linux-amd64/bmctl .
```

Comment:
I tested `gsutil cp gs://anthos-baremetal-release/bmctl/1.12.0/linux-amd64/bmctl .` and it works. 👍